site/content/integrate/plugins/community-plugin-marketplace.md

```diff
@@ -1,63 +1,63 @@
 
-Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
-
-## Requirements for Adding a Community Plugin to the Marketplace
+### Requirements for adding a community plugin to the Marketplace
 
 Every community plugin must fulfill the following checklist to be added to the Marketplace:
 
-### Product Requirements (Checked by a Product Manager)
+**Product requirements (checked by a Product Manager)**
 
 1. The plugin is published under an [Open Source license](https://opensource.org/licenses/alphabetical).
 2. The source code is available in a public Git repository.
 3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
-4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
-5. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
-6. The plugin has to be out of Beta and be released with at least v1.0.0.
-7. All configuration is accessible via the Mattermost interface.
-8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention]({{< ref "/integrate/plugins/manifest-reference#id" >}}).
+4. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
+5. The plugin has to be out of Beta and be released with at least v1.0.0.
+6. All configuration is accessible via the Mattermost interface.
+7. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
 
-### Technical Requirements (Checked by developers of the Toolkit or the Integrations team)
+**Technical requirements (checked by developers of the Toolkit team or the Integrations team)**
```

Comment:
Given that these two team don't exist any longer `Interstellar` should be used here.